idseq_dag/steps/generate_phylo_tree.py

```diff
@@ -79,24 +80,15 @@ def run(self):
         ksnp3_input_file = f"{self.output_dir_local}/inputs.txt"
         command.execute(f"cd {input_dir_for_ksnp3}/..; MakeKSNP3infile {os.path.basename(input_dir_for_ksnp3)} {ksnp3_input_file} A")
 
-        # Specify which genomes should be used for annotation.
-        # Specify the names of the genomes that should be used for annotation.
-        # Here, we use the full genomes from genbank.
+        # Specify the names of finished reference genomes.
+        # Used for annotation & variant-calling.
         annotated_genome_input = f"{self.output_dir_local}/annotated_genomes"
-        genbanbk_fasta_files = list(genbank_fastas.values())
+        genbanbk_fasta_files = list(genbank_fastas.values()) + list(accession_fastas.values())
         if genbanbk_fasta_files:
             grep_options = " ".join([f"-e '{path}'" for path in genbanbk_fasta_files])
             command.execute(f"grep {grep_options} {ksnp3_input_file} | cut -f2 > {annotated_genome_input}")
 
-        # Now run ksnp3.
-        # We can choose among 4 different output files, see http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0081760#s2:
```

Comment:
oh are these ksnp3 comments no longer valid?

Reply:
it's all in the ksnp3 manual, no point cluttering up our own code